### DISCUSSION TITLE
pi-Base community call Thu 2024 Apr 11 3pm Eastern

### DISCUSSION THREAD

**Discussion by prabau** [pi-base/data#596] [Announcements]
Agenda:

- https://github.com/pi-base/web/pull/114
- https://github.com/pi-base/data/issues/612
- https://github.com/pi-base/data/issues/626

Zoom link: https://southalabama.zoom.us/j/93833246214

**Comment by StevenClontz** [comment]
make filter on space/related full width on small viewports...

**Reply by StevenClontz** [reply]
will try web#114 out for a few weeks at least

**Comment by StevenClontz** [comment]
#612 - we will introduce just (hausdorff) continuum to start with.
